src/textarea/README.md

```diff
@@ -0,0 +1,135 @@
+# Textarea Component
+
+Component for larger text inputs.
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {StatefulTextarea} from 'baseui/textarea';
+
+ export default () => {
+  return (
+    <StatefulTextarea name="description"/>
+  );
+};
+```
+
+ ### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {StatefulTextarea, StyledTextarea} from 'baseui/textarea';
+import {withStyle} from 'styletron-react';
+
+ const TextareaWithStyle = withStyle(StyledTextarea, props => {
+  const {$isFocused, $theme: {sizing}} = props;
+  return {
+    borderRadius: $isFocused ? '0' : sizing.scale100,
+  };
+});
+
+ export default () => {
+  return (
+    <div>
+      <StatefulTextarea
+        overrides={{Input: TextareaWithStyle}}
+        onChange={() => {console.log('textarea changed')}}
+      />
+    </div>
+  );
+};
+```
+
+## Exports
+
+* `Textarea`
+* `StatefulTextarea`
+* `StatefulContainer`
+* `StyledTextareaContainer`
+* `StyledTextarea`
+* `STATE_CHANGE_TYPE`
+* `SIZE`
+
+## StatefulTextarea API
+
+* All properties of the input's StatefulContainer except `children`
+* All properties of the Textarea comonent
+
+## Stateful container API (imported from `baseui/input`)
+
+* `children?: (props: Props) => React.Node = null`
```

Comment:
This one should be `Required`. Also since it's reexported from input, maybe we can link to the Input's readme here.